packages/ast/types/src/encoding/context.d.ts

```diff
@@ -26,20 +26,20 @@ export declare class GenericParseContext implements ParseContext {
     ref: ProtoRef;
     constructor(ref: ProtoRef, store: ProtoStore, options: TelescopeOptions);
     pluginValue(name: any): any;
-    isExcluded(): boolean;
+    isExcluded(): any;
     addUtil(util: any): void;
     addImport(imp: ImportUsage): void;
     addImportDerivative(imp: DerivativeImport): void;
     getTypeNameFromFieldName(name: string, importSrc: string): string;
     getTypeName(field: ProtoField): string;
-    lookupTypeFromCurrentPath(field: ProtoField, currentProtoPath: string): import("@cosmology/proto-parser").Lookup;
```

Comment:
looks like we lost some types here, sometimes it's a build order issue, perhaps try buiding the parser and then ast.. or maybe we need to start pulling dependencies out of parser that are required by ast. The ast should not have many deps.